packages/@o3r/styling/schematics/ng-add/index.ts

```diff
@@ -30,6 +37,16 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
           parentPackageInfo: depsInfo.packageName,
           dependencyType
         }),
+        registerPackageCollectionSchematics(JSON.parse(packageJsonPath)),
+        setupSchematicsDefaultParams({
+          'component': {
```

Comment:
Is it working without specifying the collection name?
`@o3r/core:` prefix is not required?
(If so it is fine for me)